dp/skaffold.yaml

```diff
@@ -72,6 +74,8 @@ deploy:
     defaultNamespace: default
     manifests:
       - "./tools/deployment/vendor/fake_sas.yml"
+      - "./tools/deployment/vendor/elasticsearch.yml"
```

Comment:
Just a question - will orc8r and dp use the same elasticsearch?

Reply:
yes, however this one is only going to be used by DP for testing (like fake sas)